### DISCUSSION TITLE
Application not being approved

### DISCUSSION THREAD

**Discussion by 0xvelhal** [community/community#142012] [GitHub Education]
### Hi everyone,

Hey Everyone,
I am trying to have the access to the GitHub Student Developer Pack and like I have done the submissions for the 4th time now and yet it shows me the same issues, I have all the correct credentials mentioned by the organization have shared proofs of my Student ID.Yet I don't know why my application is being rejected , I tried to create a ticket for the same but it pops up saying GitHub has made kind of community for this discussions and these type of questions will be answered here.I need a solution for this.

Thank you.

**Comment by Iced-Coded** [comment]
Same thing, though, I tried applying 5 months ago and a year ago.
Also using Student ID, also going through all the "Circles of Hell", and those being: Adding billing information, changing profile's name, **_getting rejected for not translating the Student ID and then getting rejected for translating it (?!?!?!?)_**.

So yeah, good luck.
